internal/ingress/annotations/proxy/main.go

```diff
@@ -25,20 +25,21 @@ import (
 
 // Config returns the proxy timeout to use in the upstream server/s
 type Config struct {
-	BodySize          string `json:"bodySize"`
-	ConnectTimeout    int    `json:"connectTimeout"`
-	SendTimeout       int    `json:"sendTimeout"`
-	ReadTimeout       int    `json:"readTimeout"`
-	BuffersNumber     int    `json:"buffersNumber"`
-	BufferSize        string `json:"bufferSize"`
-	CookieDomain      string `json:"cookieDomain"`
-	CookiePath        string `json:"cookiePath"`
-	NextUpstream      string `json:"nextUpstream"`
-	NextUpstreamTries int    `json:"nextUpstreamTries"`
-	ProxyRedirectFrom string `json:"proxyRedirectFrom"`
-	ProxyRedirectTo   string `json:"proxyRedirectTo"`
-	RequestBuffering  string `json:"requestBuffering"`
-	ProxyBuffering    string `json:"proxyBuffering"`
+	BodySize            string `json:"bodySize"`
+	ConnectTimeout      int    `json:"connectTimeout"`
+	SendTimeout         int    `json:"sendTimeout"`
+	ReadTimeout         int    `json:"readTimeout"`
+	BuffersNumber       int    `json:"buffersNumber"`
+	BufferSize          string `json:"bufferSize"`
+	CookieDomain        string `json:"cookieDomain"`
+	CookiePath          string `json:"cookiePath"`
+	NextUpstream        string `json:"nextUpstream"`
+	NextUpstreamTimeout string `json:"nextUpstreamTimeout"`
```

Comment:
Why `string`? Should be `int`.

Reply:
It's a `time` value according to the docs, so it takes values like `5s`, `3m`, etc